### DISCUSSION TITLE
how to go on testing when assert failed ？

### DISCUSSION THREAD

**Discussion by sinck97** [cypress-io/cypress#15216] [Questions and Help] [answered]
I need to judge element exist or not, in if ( )
when exist I can judge as followed
cy.get('.A').then($el)=>{
const len = $el.length
if ( len == 1){
             ……
}
}

but when the element not exist
assert will fail

**Comment by bahmutov** [comment, marked as answer]
See https://on.cypress.io/conditional-testing
